src/docfx/build/xref/XrefMap.cs

```diff
@@ -354,21 +358,105 @@ private static (List<Error> errors, InternalXrefSpec spec) LoadSchemaDocument(Co
             }
 
             var errors = new List<Error>();
-            var (schemaErrors, content) = JsonUtility.ToObjectWithSchemaValidation(
+            var (schemaErrors, _) = JsonUtility.ToObjectWithSchemaValidation(
                 obj,
                 schema.Type,
                 transform: AttributeTransformer.TransformXref(context, file, null, extensionData));
 
             errors.AddRange(schemaErrors);
-            var xref = new InternalXrefSpec
+
+            var extensionDataByUid = new Dictionary<string, (bool isRoot, Dictionary<string, Lazy<JValue>> properties)>();
+
+            if (extensionData.Count == 0)
```

Comment:
nit: no need to special case `extensionData.Count == 0` here, can just loop over it.